src/PrestaShopBundle/Controller/Admin/Sell/Catalog/AttributeController.php

```diff
@@ -155,14 +179,43 @@ public function createAction(int $attributeGroupId)
      * @param int $attributeId
      * @param int $attributeGroupId
      *
-     * @return RedirectResponse
+     * @return Response
      */
-    public function editAction(int $attributeId, int $attributeGroupId)
+    public function editAction(Request $request, int $attributeId, int $attributeGroupId): Response
     {
-        // @todo: implement in another pr
-        return $this->redirectToRoute('admin_attributes_index', [
-            'attributeGroupId' => $attributeGroupId,
-        ]);
+        $attributeFormBuilder = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\AttributeFormBuilder');
+        $attributeFormHandler = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Handler\AttributeFormHandler');
```

Comment:
Same suggestion you should use `::class` instead here

Reply:
(same as above)